boot/freeldr/freeldr/lib/peloader.c

```diff
@@ -144,6 +147,7 @@ WinLdrScanImportDescriptorTable(IN OUT PLIST_ENTRY ModuleListHead,
             Success = WinLdrpLoadAndScanReferencedDll(ModuleListHead,
                                                       DirectoryPath,
                                                       ImportName,
+                                                      &ScanDTE->InLoadOrderLinks,
```

Comment:
Only `InLoadOrderLinks` is used?
We can safely ignore `InMemoryOrderLinks` and `InInitializationOrderLinks` ?

Reply:
Yes - they don't seem to be touched by the bootloader. NT uses InLoadOrderLinks to loop through the list, and calls DllInitialize when it's available. Without this patch, netio.sys (I think it is) fails because msrpc.sys hasn't been initialized yet.